apps/charterafrica/src/pages/api/v1/resources/[action].page.js

```diff
@@ -0,0 +1,29 @@
+import {
+  createEcosystemResource,
+  updateEcosystemResource,
+} from "@/charterafrica/lib/tools/syncEcosystem";
+
+const isApiKeyValid = (key) => {
+  return key && key === process.env.LOCAL_API_KEY;
+};
+
+const functionMap = {
+  "create-ecosystem-resource": createEcosystemResource,
+  "update-ecosystem-resource": updateEcosystemResource,
```

Comment:
```suggestion
  "update-ecosystem-list": updateEcosystemList,
  "update-ecosystem-content": updateEcosystemContent,
```

I think this clarifies what were doing: Updating list of ecosystem resources and fetching actual content after that based on the list.